### PR TITLE
Minutes from meeting on 05.04.2018

### PR DESCRIPTION
**Pull request by daneos**
Here are the minutes. If I forgot about something don't hesitate to add it.
@reszelaz, add your slides :)

### CODE REVIEW

**Comment by reszelaz** [comment]
Impressive speed:) looks very complete to me ...tomorrow I will review my notes to see if there is sth more to add, and of course I'll upload the slide :)

**Comment by teresanunez** [comment]
Hi, for me it is fine like it is. Thanks.

**Comment by teresanunez** [comment]
@reszelaz, if we decide to make the Sardana WS in Prague, we have to check first with
 the people from ELI if we could have a room for it, don't we?. Would you take care of that?.

**Comment by guifrecuni** [comment]
Hi, just to let you know that I've received an email from the organizers that having a Sardana meeting in parallel with the TANGO EC Meeting would be possible at the venue.

**Comment by reszelaz** [comment]
I added the slides. For me it can be merged.

File: 20180405/MINUTES.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+		* Sardana organization would host only the 3rd party plugin register
+		* Everyone could use repositories layout of their choice
+		* Possible approach: single controller repos
+			* Pros:
+				* easier documentation
+				* allows better code structure
+				* fits better for CI
+			* Cons:
+				* you need to install multiple repos/packages instead of just one
+			* See https://github.com/amilan/lib-maxiv-sardana_albaem
```
Comment (amilan) [inline 20180405/MINUTES.md]: The link is pointing to the master branch, where unfortunately I haven't merge the refactoring branch, therefore, it's better to point to this url:
https://github.com/amilan/lib-maxiv-sardana_albaem/tree/refactoring

**Comment by teresanunez** [comment]
It doesn't really belong here ... but to complete the info about the RunningMacros attribute, the information contained there is also in the Status attribute of the Door device. So I will try the people at DESY to get it from the Status and remove this attribute from our private version.

File: 20180405/MINUTES.md (hunk 1 above)
Comment (reszelaz) [reply 20180405/MINUTES.md]: Thanks for pointing this out. I'm changing ti.

**Comment by reszelaz** [comment]
I think there will be no more corrections. Merging..

**Pull request merged**